demo-shell/src/app/components/form/form.component.scss

```diff
@@ -31,8 +31,10 @@
     }
 }
 
-.app-form-config-editor {
-    height: 500px !important;
+.mat-tab-body {
```

Comment:
Please do not refer to internal Material classes

Reply:
Done